internal/parser/grammar/IMAPParser.g4

```diff
@@ -4,8 +4,12 @@ options {
 	tokenVocab = IMAPLexer;
 }
 
+@members {
+    virtual std::string consumeTokens(std::string) = 0;
+}
+
 // 2.2. Commands and Responses
-crlf: CR LF;
+crlf: CR? LF;
```

Comment:
:scream: why? shouldn't CRLF _always_ be CR and LF together?

Reply:
Woops, forgot to revert this :sweat_smile: